tests/.infra/centos-ci/rc/cico-rc-multiuser-happy-path-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+set -e
+set +x
```

Comment:
Doesn't it `set +x` by default?

Reply:
you can get current set status by command `echo ${-}`

Reply:
I wanted to indicate obviously it and to be sure.